src/profile-logic/profile-data.js

```diff
@@ -1507,6 +1507,47 @@ export function isNetworkMarker(marker: TracingMarker): boolean {
   return !!(marker.data && marker.data.type === 'Network');
 }
 
+export function getNetworkMarkers(thread: Thread): Array<NetworkPayload> {
```

Comment:
I think my preference after all the marker work of last week would be to return `TracingMarker[]` here, instead of just the payload. If that proves difficult, I'd be fine doing that as a follow-up, as our types are not very rich here yet.

Reply:
👍